react/components/ProductImages/components/HighQualityProductImage.tsx

```diff
@@ -94,24 +101,27 @@ function HighQualityProductImage(props: Props) {
             }}
             // See comment regarding sizes below
             sizes="(max-width: 64.1rem) 100vw, 50vw"
-            src={imageUrl(
-              src,
-              DEFAULT_SIZE * zoomFactor,
-              MAX_SIZE,
-              aspectRatio
-            )}
+            src={
+              imageSizes
+                ? imageUrl(src, defaultSize * zoomFactor, maxSize, aspectRatio)
+                : src
+            }
           />
         }
       >
         <img
           alt={alt}
           title={alt}
           loading="lazy"
-          srcSet={srcSet}
+          srcSet={imageSizes ? srcSet : undefined}
```

Comment:
`srcSet` will be `undefined` if `imageSizes` is falsy, right? 

```suggestion
          srcSet={srcSet}
```